Backend/server.js

```diff
@@ -0,0 +1,93 @@
+import express from 'express';
+import session from 'express-session';
+import cors from 'cors';
+import morgan from 'morgan';
+import { sequelize } from './database.js';
+import { User, Mentor, Mentee } from './models/index.js';
+import userRoutes from "./Routes/users.js";
+import SequelizeStoreInit from 'connect-session-sequelize';
+import { PrismaClient } from '@prisma/client';
+
+const app = express();
+
+app.use(cors({
+  origin: 'http://localhost:5173',
+  credentials: true
+}));
+app.use(express.json()); // Middleware for parsing JSON bodies from HTTP requests
+app.use(morgan("combined"))
+
+const SequelizeStore = SequelizeStoreInit(session.Store);
+const sessionStore = new SequelizeStore({
+  db: sequelize
+});
+
+// Session middleware
+app.use(
+  session({
+    secret: 'your-secret-key',
+    resave: false,
+    saveUninitialized: false,
+    store: sessionStore,
+    cookie: {
+      sameSite: false,
+      secure: false,
+      expires: new Date(Date.now() + (365 * 24 * 60 * 60 * 1000)) // 1 year in milliseconds
+    }
+  })
+);
+sessionStore.sync();
+
+app.use(userRoutes);
+
+// Route to get all posts, with associated users
+app.get('/posts', async (req, res) => {
```

Comment:
Do we have the feature for posts? If not we can remove them?

Reply:
Resolved